client.lua

```diff
@@ -47,12 +47,17 @@ Citizen.CreateThread(function()
 		Citizen.Wait(0)
 		local playerPed = PlayerPedId()
 		local coords = GetEntityCoords(playerPed)
+		local canSleep = true
```

Comment:
You never set canSleep to `true` somewhere in this function.

Reply:
Forgot the canSleep = false inside GetDistance, the placement og canSleep = true is right just missing the counter var ^^